xhistogram/test/fixtures.py

```diff
@@ -12,3 +15,33 @@ def raise_if_computed():
         a = a.rechunk(chunks)
 
     return a
+
+
+@pytest.fixture(scope="module")
+def dataarray_factory():
+    def _dataarray_factory(shape=(5, 20)):
+        data = np.random.randn(*shape)
+        dims = ["x", "y", "z", "w"][: len(shape)]
+        da = xr.DataArray(data, dims=dims, name="T")
+        return da
+
+    return _dataarray_factory
+
+
+@pytest.fixture(scope="module")
+def dataset_factory():
+    """Random dataset with every variable having the same shape"""
+
+    def _dataset_factory(ndim=2, n_vars=2):
```

Comment:
Nitpick: it might be good to standardize whether or not we use underscores in `n*` variable names. I don't have a strong option on if we should use, for example, `n_vars` or `nvars`, but picking and sticking with a convention would be nice.